client/src/modules/search-results/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Card, Grid, Typography } from '@material-ui/core';
+import Layout from '../home/shared/components/Layout';
+
+const SearchResults: React.FC = () => {
+  // TODO: Replace with `useEventsQuery()`!
+  const data = {
+    searchResults: [],
+    searchTerm: 'London',
+  };
+
+  return (
+    <Layout>
+      <h1>
```

Comment:
If we're going with MUI components, this should be a Typography component with variant/component h1

Reply:
Will do.